src/auth/src/error.rs

```diff
@@ -38,6 +38,14 @@ pub enum Error {
         location: Location,
     },
 
+    #[snafu(display("Failed to convert to utf8, error={:?}", error))]
```

Comment:
```suggestion
    #[snafu(display("Failed to convert to UTF-8"))]
```

We don't need to print the error manually, the stack should print itself to the root source.